### ISSUE TITLE
Cannot instantiate Desktop() object in windows

### ISSUE THREAD

**Issue by amrit-poudel** [pyansys/pyaedt#679]
I have installed pyaedt, pythonnet in a windows machine with python v 3.7. However, I can't seem to run the following simple command without an assertion error:
```
>>> from pyaedt import Desktop
>>> d = Desktop(specified_version="2016.1", non_graphical=False). 
```
AssertionError: Specified version 2016.1 not known.

I do not have any environment variables set, and Ansys binary/executable is not in the path. 

I'd appreciate if someone could provide a pointer or two on how to instantiate this object and make the Ansys GUI pop up after executing the command above

I am not sure whether simply specifying "version number" is enough to open the Ansys GUI when using pyaedt. I thought one may also need to specify the path to the Ansys executable, but the Desktop() object does not seem to take the path to the executable as an input.

Thanks for any help!

**Comment by MaxJPRey** [comment]
Hi @amrit-poudel,

Based on the error you got, I am assuming that you are working with ANSYS EM Suite 2016.1.
PyAEDT requires ANSYS EM Suite 2021 R1 or later (see https://aedtdocs.pyansys.com/Resources/Installation.html).

I hope it helps you.
Thanks.

**Comment by amrit-poudel** [comment]
@MaxJPRey Thanks for that note. The following link says PyAEDT supports prior versions of AEDT, including 2021. 

https://aedtdocs.pyansys.com/

Please see the dependency section. I am a bit confused.

**Comment by akaszynski** [comment]
> @MaxJPRey Thanks for that note. The following link says PyAEDT supports prior versions of AEDT, including 2021.

@MaxJPRey, let's update the documentation and provide either a table or list of supported Ansys AEDT releases to clarify this. I agree that it's not clear.

**Comment by MaxJPRey** [comment]
The documentation has been improved.
In addition we now provide the user with some information depending on the release version he provides when instantiating a new PyAEDT `Desktop` object.
Work merged in #708 .
